### PR TITLE
JSMpeg options + key event defaults prevented

### PR DESCRIPTION
**Pull request by lnneth**

### CODE REVIEW

**Comment by jeda** [comment]
Should make the bitrate to match and will also now catch more keys.

**Comment by Simonx22** [comment]
This currently breaks input. Commenting out ``event.preventDefault()`` for the mouse inputs seems to fix it apparently. Still looking into which event is causing this.

File: components/player/viewer.vue [hunk 1]
```diff
@@ -86,35 +88,41 @@
...
             },
 
             didMouseMove(event) {
+		event.preventDefault()
                 const { x, y } = this.calculatePos(event)
 
                 this.emitEvent({ x, y }, 'MOUSE_MOVE')
             },
             didMouseDown(event) {
+		event.preventDefault()
```
Comment (Simonx22) [inline components/player/viewer.vue]: Remove this line

File: components/player/viewer.vue [hunk 2]
```diff
@@ -86,35 +88,41 @@
...
                 const { button } = event,
                     { x, y } = this.calculatePos(event)
 
                 this.emitEvent({ x, y, button: button + 1 }, 'MOUSE_DOWN')
             },
             didMouseUp(event) {
+		event.preventDefault()
```
Comment (Simonx22) [inline components/player/viewer.vue]: And this one

**Comment by Simonx22** [comment]
Remove these two lines and the keys will work. Other than that: LGTM! Pressing CTRL no longer breaks the input inside the VM.

**Review by Simonx22 — CHANGES_REQUESTED** [review]

**Comment by Simonx22** [comment]
Current state of this PR:
- it fixes the keys being stuck bug
- scrolling on Firefox is broken but it's not related to this PR

**Comment by jeda** [comment]
I will merge this for now and make an issue for the scrolling bug. At least this is a fix that will help some, and I'd like this being there for now. Thanks @lnneth for the fix!

**Pull request merged**